framework_tvb/tvb/interfaces/rest/server/rest_api.py

```diff
@@ -28,7 +28,7 @@
 #
 #
 
-from flask_restful import Api
+from flask_restplus import Api
```

Comment:
Are you certain there are only 2 imports ?
We should make sure we do not depend on both libraries

Reply:
Yes, there are only 2 imports.